src/Forms/GridField/GridFieldFilterHeader.php

```diff
@@ -491,4 +505,36 @@ private function getTitle(object $inst): string
 
         return ClassInfo::shortName($inst);
     }
+
+    /**
+     * Transform search contex into basic search context (preserve all releavnt search settings)
+     *
+     * @param GridField $gridField
+     * @param SearchContext $searchContext
+     * @return BasicSearchContext
```

Comment:
```suggestion
```
Don't need PHPDoc typing when we use actual typing in the method signature itself.